### PR TITLE
Clone to a local filesystem (tmp), instead of memory

### PR DESCRIPTION
**Pull request by clickyotomy**
Possible fix for https://github.com/samber/git-contrib-graph/issues/5.

### CODE REVIEW

File: git-contrib-graph.go [hunk 1]
```diff
@@ -205,7 +205,11 @@ func getRepo(git_path string, git_remote string) object.CommitIter {
 		repo, err = git.PlainOpen(git_path)
 	} else if git_remote != "" {
 		path = git_remote
-		repo, err = git.Clone(memory.NewStorage(), nil, &git.CloneOptions{
+		dir, err := ioutil.TempDir("", git_path)
```
Comment (samber) [inline git-contrib-graph.go]: do you mean `ioutil.TempDir("", git_remote)` ? ;)

**Review by samber — CHANGES_REQUESTED** [review]
this is a nice job @clickyotomy ! 💪 

a small fix and i'm ready for a merge ;)

**Comment by clickyotomy** [comment]
Thank you, @samber! :)
I have made a change to add a prefix for `TempDir` function.
During runtime, it would clone the repo into a temporary directory, such as this:
```ShellSession
$ ./git-contrib-graph --git-remote https://github.com/samber/invoice-as-a-service --full-graph --interval month
/var/folders/pt/n0_dbhkj33x_rd_p1nt4s23w0000gn/T/invoice-as-a-service110657057
```
Would that be OK?

**Comment by samber** [comment]
The `split` is great !

Let's merge that 👍

**Review by samber — APPROVED** [review]

**Pull request merged**